classes/module/Module.php

```diff
@@ -3297,10 +3334,39 @@ public function removeOverride($classname)
             }
 
             $to_delete = preg_match('/<\?(?:php)?\s+(?:abstract|interface)?\s*?class\s+' . $classname . '\s+extends\s+' . $classname . 'Core\s*?[{]\s*?[}]/ism', $code);
+
+            if (!$to_delete) {
+                // Check if override class has remaining code
+                eval(
```

Comment:
Can you add some explanation on this complicated thing! 😅

```
To detect if the class has remaining code we dynamically create an class which contains the remaining code, the we use ReflectionClass to analyze what this code actually contains If no valuable code remains then we can delete it
```

Reply:
It would have been nice to have this kind of feature externalized in a dedicated class but it might be out of this PR's scope!

Reply:
Congratulations on this idea however, it's a bit ugly but yet it's smart 😅

Reply:
Agreed with @jolelievre : you made this already huge class a little bit more large 😅 

Nice-to-have: put this logic somewhere else